Libraries/LibJS/Tests/labels.js

```diff
@@ -1,20 +1,22 @@
-load("test-common.js");
-
-try {
+test("labeled plain scope", () => {
     test: {
         let o = 1;
-        assert(o === 1);
+        expect(o).toBe(1);
```

Comment:
In other files you're using `expect.fail()`, let's stick with one :)

Reply:
Whoops, wrong line. A bit below (it's `expect().fail()`)

Reply:
Oops, `expect.fail()` is wrong one. Nice catch